README.md

```diff
@@ -93,37 +93,35 @@ properly format your code.
 Apache 2.0; see [`LICENSE`](LICENSE) for details.
 
 <!-- Repo links -->
+
 [ff_dart]: https://github.com/GoogleCloudPlatform/functions-framework-dart
 
 <!-- Unit Test links -->
-[ff_dart_unit_img]:
-https://github.com/GoogleCloudPlatform/functions-framework-dart/workflows/Dart%20Unit%20CI/badge.svg
-[ff_dart_unit_link]:
-https://github.com/GoogleCloudPlatform/functions-framework-dart/actions?query=workflow%3A"Dart+Unit+CI"
+
+[ff_dart_unit_img]: https://github.com/GoogleCloudPlatform/functions-framework-dart/workflows/Dart%20Unit%20CI/badge.svg
```

Comment:
Due to the space after the colon, this fails the line length limit. When we split across two lines, it's ok that the URL itself is too long (won't cause a lint error).

Reply:
Done 🚀
all needed splits are done with now extra trailing space and the linter returned 0 errors!